xblock/run_script.py

```diff
@@ -13,6 +20,6 @@ def run_script(pycode):
 
     # execute it.
     globs = {}
-    exec pycode in globs, globs  # pylint: disable=W0122
+    exec(pycode, globs, globs)  # pylint: disable=W0122
```

Comment:
Hmm.  This must be untested.  I think this would break in python2.  It should be using the six version of exec.